lib/view_model/edit_backup_password_view_model.dart

```diff
@@ -37,8 +38,6 @@ abstract class EditBackupPasswordViewModelBase with Store {
   @action
   Future<void> save() async {
     final key = generateStoreKeyFor(key: SecretStoreKey.backupPassword);
-    await secureStorage.delete(key: key);
-    await secureStorage.write(key: key, value: backupPassword);
-    secretStore.write(key: key, value: backupPassword);
```

Comment:
this shouldn't be deleted, and please double check if any side-effect like this was made unintentionally through the changes